backend/models/question.py

```diff
@@ -4,29 +4,32 @@
 
 from backend.constants import QUESTION_TYPES, REQUIRED_QUESTION_TYPE_DATA
 
-_TESTS_TYPE = t.Union[t.Dict[str, str], int]
+_TESTS_TYPE = dict[str, str] | int
 
 
 class Unittests(BaseModel):
     """Schema model for unittest suites in code questions."""
+
     allow_failure: bool = False
     tests: _TESTS_TYPE
 
     @validator("tests")
-    def validate_tests(cls, value: _TESTS_TYPE) -> _TESTS_TYPE:
+    def validate_tests(cls, value: _TESTS_TYPE) -> _TESTS_TYPE | None:
```

Comment:
How can this return `None` is `value` is `dict[str, str] | int`?
The only other exit path I see from this function is raising an error.